### PR TITLE
mute syncing YAy

### PR DESCRIPTION
**Pull request by Fury-101**
@ElijahRus250 make sure to enable member intents and presence intents in dev portal
doesn't work for timing i'll do that later
if u let the mute run out of time the person will stay muted in the discord 4ever so u should probably not merge rn

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -465,12 +466,11 @@
...
-        // let displayNickname = `${guild_mute_username}`;
-        // let serverMembers = client.guilds.cache.get(serverID).members
-        // let matchedMember = serverMembers.cache.find(m => m.displayName === displayNickname);
-        // if (!matchedMember) {return}
-        // matchedMember.roles.add('529453283782164502')
+        let displayNickname = guild_mute_username;
+        let serverMembers = client.guilds.cache.get(serverID).members.cache;
+        let matchedMember = serverMembers.findKey(user => user.nickname == displayNickname);
+        if (!matchedMember) {return}
+        serverMembers.get(matchedMember).roles.add('529453283782164502');
```
Comment (ElijahRus250) [inline index.js:473]: Please make sure this function will work as I am pretty sure you need to get a role from cache

**Review by ElijahRus250 — COMMENTED** [review]
We should do every 30 or so minutes poll the api, check if they are muted, then find the user, make sure they have the role and if they are not muted in game remove the role

File: index.js (hunk 1 above)
Comment (Fury-101) [reply index.js:473]: it does - just tested it

**Comment by Fury-101** [comment]
u can probs merge now that mute timing works but there's no api request yet

**Pull request merged**